src/com/owncloud/android/authentication/AuthenticatorActivity.java

```diff
@@ -1143,6 +1145,17 @@ private String normalizeUrlSuffix(String url) {
         return url;
     }
 
+    private String stripIndexPhpOrAppsFiles(String url, EditText mHostUrlInput) {
+        if (url.endsWith("/index.php")) {

```

Comment:
What about: http://hostname/index.php/ this is valid URL and will probably fail?